src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -91,19 +151,56 @@ const ikoneForJournalposttype: Record<Journalposttype, React.ReactElement> = {
 };
 
 const Dokumenter: React.FC<{ fagsakPerson: IFagsakPerson }> = ({ fagsakPerson }) => {
+    const [vedleggRequest, settVedleggRequest] = useState<VedleggRequest>({
+        fagsakPersonId: fagsakPerson.id,
+    });
+
+    const settVedlegg = (key: keyof VedleggRequest) => {
+        return (val?: string | number) =>
+            settVedleggRequest((prevState: VedleggRequest) =>
+                oppdaterVedleggFilter(prevState, key, val)
+            );
+    };
+
     const dokumentConfig: AxiosRequestConfig = useMemo(
         () => ({
-            method: 'GET',
-            url: `/familie-ef-sak/api/vedlegg/fagsak-person/${fagsakPerson.id}`,
+            method: 'POST',
+            url: `/familie-ef-sak/api/vedlegg/fagsak-person`,
+            data: vedleggRequest,
         }),
-        [fagsakPerson]
+        [vedleggRequest]
     );
 
     const dokumentResponse = useDataHenter<Dokumentinfo[], null>(dokumentConfig);
+    const { toggles } = useToggles();
+    const [visFeilregistrerteOgAvbruttValgt, setVisFeilregistrerteOgAvbruttValgt] =
+        React.useState(false);
 
-    const dokumentGruppeSkalIkkeVises = (dokumenter: Dokumentinfo[]): boolean => {
+    const visFeilregistrerteOgAvbrutt = () => {
+        setVisFeilregistrerteOgAvbruttValgt(!visFeilregistrerteOgAvbruttValgt);
+    };
+
+    const dokumentGruppeSkalVises = (dokumenter: Dokumentinfo[]): boolean => {
```

Comment:
Kunne kanskje flyttes til `Dokumentoversikt/utils.ts` for å gjøre denne filen lettere å lese

Reply:
Også mulig jeg ikke helt skjønte bruken av denne, men er ikke dette tilsvarende: 
```
const dokumentGruppeSkalVises = (dokumenter: Dokumentinfo[], visFeilregistrerteOgAvbrutte: boolean) =>
        dokumenter.some((dokument) =>
            [Journalstatus.FEILREGISTRERT, Journalstatus.AVBRUTT].includes(dokument.journalstatus)
        )
            ? visFeilregistrerteOgAvbrutte
            : true;
```

Hvis dokumentene ikke har status feilregistrert/avbrutt har de vel en av de andre default-statuser.

Reply:
Foreslått måte viser feilregistrerte i tillegg til de andre, men det er ønskelig at hvis man krysser av for feilregistrerte/avbrutt skal kun de vises. Men er helt sikkert en mer elegant måte å gjøre det på enn det som jeg gjorde her 😅

Reply:
Ah, skjønner. Blir dette kanskje riktig da:

```
    const dokumentGruppeSkalVises = (
        dokumenter: Dokumentinfo[],
        visFeilregistrerteOgAvbrutte: boolean
    ) => {
        const harFeilregistrerteEllerAvbrutte = dokumenter.some((dokument) =>
            [Journalstatus.FEILREGISTRERT, Journalstatus.AVBRUTT].includes(dokument.journalstatus)
        );

        return visFeilregistrerteOgAvbrutte
            ? harFeilregistrerteEllerAvbrutte
            : !harFeilregistrerteEllerAvbrutte;
    };
```

Reply:
Takk, det funket!